interface-definitions/vpn-openconnect.xml.in

```diff
@@ -50,6 +50,16 @@
                   </leafNode>
                 </children>
               </node>
+              <leafNode name="groups">
```

Comment:
Could we rename it to `group`? Even if it has "multiple" values, it should be just "group"
For example, as we have it in the firewall, `set firewall group xxx`
We have only a couple of exceptions like `interfaces` and `protocols`

Reply:
Roger